docs/fiddles/features/drag-and-drop/main.js

```diff
@@ -1,20 +1,23 @@
 const { app, BrowserWindow, ipcMain, nativeImage, NativeImage } = require('electron')
-const fs = require('fs');
-const http = require('http');
+const path = require('path')
+const fs = require('fs')
+const http = require('http')
```

Comment:
Question: should we update this to `https`? 🤔

Reply:
I don't know how to answer this 😆. I don't think we *need* to? But I will because it works?